web/src/pages/User/Login.tsx

```diff
@@ -74,13 +75,19 @@ const Page: React.FC = () => {
     }
   }
 
+
   if (localStorage.getItem('token')) {
     history.replace('/');
     return null;
   }
   return (
     <div className={styles.container}>
       <div className={styles.lang}>
+
+        <div className={styles.github} >
+          <a className="githubIcon" href="https://apisix.apache.org"><GithubOutlined /></a> 
+        </div>
```

Comment:
Also, the link maybe can add `target='_blank'` to let the user can open the page in the new tab.